Templating/FilterTrait.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Liip\ImagineBundle\Templating;
 
+@trigger_error('The ' . FilterTrait::class . ' trait is deprecated since version 2.7 and will be removed in 3.0; use Twig instead.', E_USER_DEPRECATED);
```

Comment:
we should also add `@deprecated` class annotations for both I think

Reply:
I followed what was done in the other files, but I agree with you.